site-src/index.md

```diff
@@ -94,8 +94,32 @@ This flexibility allows the API to adapt to vastly different
 organizational models and implementations while remaining a portable and
 standard API.
 
+## What's the difference between Gateway API and an API Gateway?
 
-## Who is working on Gateway?
+The concept of a Gateway is a networking component that allows data to transfer from one
+network to another network.
+
+An API Gateway is a general concept that describes anything that exposes capabilities of a backend service, while providing extra capabilities for traffic routing and manipulation, such as load balancing, request and response transformation, and sometimes more advanced features like authentication and authorization, rate limiting, and circuit breaking.
+
+Most Gateway API implementations are API Gateways to some extent, but not all API Gateways are Gateway API implementations.
```

Comment:
Tiny nit: Do you mind wrapping these lines at 80 chars to match the rest of the file? (I may be off on the number here, just looking for shorter lines that match the rest of the file).